l5kit/l5kit/rasterization/rasterizer_builder.py

```diff
@@ -65,9 +73,27 @@ def build_rasterizer(cfg: dict, data_manager: DataManager) -> Rasterizer:
     history_num_frames = cfg["model_params"]["history_num_frames"]
 
     if map_type in ["py_satellite", "satellite_rgb"]:
-        sat_image, meta = _load_image_and_metadata(raster_cfg["satellite_map_key"], data_manager)
-        ecef_to_sat = np.array(meta["ecef_to_image"], dtype=np.float64)
-        pose_to_ecef = load_pose_to_ecef()
+        sat_image_key = raster_cfg["satellite_map_key"]
+        sat_image = _load_satellite_map(sat_image_key, data_manager)
+
+        sat_meta_key = os.path.splitext(sat_image_key)[0] + ".json"
```

Comment:
Why do we do this? shouldn't the suffix be already in config?

Reply:
this is not the `meta.json` for the dataset (pose_to_ecef), is the meta for the satellite map (ecef_to_image, where image is satellite space) which has the same name of the map but different extension. That's another thing to solve xD

Reply:
Oh, I see, yes, let's make the same change there please!